library/common/network/BUILD

```diff
@@ -6,9 +6,21 @@ envoy_package()
 
 envoy_cc_library(
     name = "mobile_utility_lib",
-    srcs = ["mobile_utility.cc"],
+    srcs = [
+        "mobile_utility.cc",
+    ] + select({
+        "//bazel:include_ifaddrs": [
+            "//third_party:android/ifaddrs-android.h",
+            "//third_party:android/LocalArray.h",
+            "//third_party:android/ScopedFd.h",
+        ],
+        "//conditions:default": [],
+    }),
     hdrs = ["mobile_utility.h"],
-    copts = ["-Ithird_party/android"],
```

Comment:
don't still need this one?

Reply:
Yeah, it seems to only really be needed when your include paths are something funky that you can't change.